controllers/config/base/controllersconfig/cf_k8s_controllers_config.yaml

```diff
@@ -1,4 +1,6 @@
 kpackImageTag: gcr.io/cf-relint-greengrass/cf-k8s-controllers/kpack/beta
 cfProcessDefaults:
   memoryMB: 500
-  diskQuotaMB: 512
\ No newline at end of file
+  diskQuotaMB: 512
+cfk8s_controller_namespace: cf-k8s-controllers-system
+#workloads_tls_secret_name: cf-k8s-workloads-ingress-cert
```

Comment:
For some reason when I run these templates as part of make deploy, my kustomize output is keeping the commented out yaml string in the generated config map.